src/libostree/ostree-gpg-verify-result.c

```diff
@@ -513,6 +513,49 @@ ostree_gpg_verify_result_describe (OstreeGpgVerifyResult *result,
   ostree_gpg_verify_result_describe_variant (variant, output_buffer, line_prefix, flags);
 }
 
+static void
+append_expire_info (GString *output_buffer,
+                    const gchar *line_prefix,
+                    const gchar *exp_type,
+                    gint64 exp_timestamp,
+                    gboolean expired)
+{
+  g_autoptr(GDateTime) date_time_utc = NULL;
+  g_autoptr(GDateTime) date_time_local = NULL;
```

Comment:
General preference in ostree is for declare-and-init, i.e. C99 not C89.

Reply:
Copy and pasting there. I'll fix that for the next thing.